eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/Program.cs

```diff
@@ -82,69 +79,66 @@ public Task ProcessEventsAsync(IEnumerable<EventData> events)
             var maxLatency = listTS.Max().TotalMilliseconds;
             var avgLatency = Math.Round(listTS.Average(ts => ts.TotalMilliseconds), 0);
 
-            lock(_lock) {
+            lock (_lock)
+            {
                 Console.Write($"[{this._partitionId}] Received {eventCount} events in {events.Count()} batch(es).");
-                Console.Write($"\tBatch (From/To): {batchFrom.ToString("HH:mm:ss.ffffff")}/{batchTo.ToString("HH:mm:ss.ffffff")}");                
+                Console.Write($"\tBatch (From/To): {batchFrom.ToString("HH:mm:ss.ffffff")}/{batchTo.ToString("HH:mm:ss.ffffff")}");
                 Console.Write($"\tElapsed msec (Min/Max/Avg): {minLatency}/{maxLatency}/{avgLatency}");
                 Console.WriteLine();
 
                 _csvOutput.WriteLine($"{this._partitionId},{events.Count()},{eventCount},{batchFrom.ToString("o")},{batchTo.ToString("o")},{minLatency},{maxLatency},{avgLatency}");
             }
-        
+
             return Task.CompletedTask;
         }
     }
 
-
     public class EventReceiver
```

Comment:
I really don't think this class offers any value; if we convert to iterator use to `ReadEventsFromPartition` then 90% of the code is gone.   I'd recommend removing it to reduce complexity.